### PR TITLE
Clean and expose r

### PR DESCRIPTION
**Pull request by duncan-clark**
package builds cleanly

### CODE REVIEW

File: src/Model.h [hunk 1]
```diff
@@ -514,21 +514,58 @@ class Model : public ShallowCopyable{
 			offsets[k]->vDyadUpdate(*net, from, to);
 		}
 	}
-
+    
+    void dyadUpdateR(int from, int to){
```
Comment (ifellows) [inline src/Model.h:518]: not to be pedantic, but what happens if from or to are less than 1?

File: src/Model.h [hunk 2]
```diff
@@ -514,21 +514,58 @@ class Model : public ShallowCopyable{
...
+        if(from > net->size() || to > net->size())
+            ::Rf_error("one of the vertex indices in the dyad update is bigger than the size of the network");
+        this->dyadUpdateR((from-1),(to-1));
```
Comment (ifellows) [inline src/Model.h:521]: do you mean to call dyadUpdate not dyadUpdateR?

**Review by ifellows — CHANGES_REQUESTED** [review]

**Comment by duncan-clark** [comment]
rebased so can automerge

**Comment by ifellows** [comment]
So ready for merge?

**Pull request merged**